docs/Examples/MultipartUpload.md

```diff
@@ -0,0 +1,57 @@
+# Multipart upload
+
+Suppose we want to upload a GIF with additional parameters in one request. How to approach this problem? It depends if our parameters should be in body of the request or in the URL.
```

Comment:
> How do we approach this problem? It depends if the parameters should be part of the body, or part of the query string

You could change to `part of the URL`, but query string is actually the more correct term IMO

Reply:
You are correct that `query string` is a better term, but it might not be more understandable vs just using URL term. But I agree with the rest of the changes here 👍